tarantool/connection.py

```diff
@@ -785,7 +793,7 @@ def _send_request_wo_reconnect(self, request):
         """
 
         assert isinstance(request, Request)
-
+        
```

Comment:
This change is both excessive and unwanted

Reply:
Done, deleted